tests/glsl-intrinsic/fragment-processing/fragment-processing-interpolate-simple.slang

```diff
@@ -0,0 +1,27 @@
+//TEST:SIMPLE(filecheck=CHECK_GLSL): -target glsl -stage fragment -entry main -allow-glsl
+//TEST:SIMPLE(filecheck=CHECK_SPV): -target spirv -emit-spirv-directly -stage fragment -entry main -allow-glsl
+#version 450
+// CHECK_SPV-DAG: OpEntryPoint
+// CHECK_GLSL-DAG: void main(
+
+layout(location = 0) out ivec4 outColorActual;
+
+layout (location = 0) in float inDataV1;
+
+bool isOk(float a, float b)   { return (a == b); }
+
+bool testFragmentProcessingInterpolateFunctions()
+{
+    float tmpStore = inDataV1;
+    return true
+        && interpolateAtCentroid(inDataV1) != -1.0f
```

Comment:
If I understood correctly, "inDataV1" is a uniform variable which means its value will be same for every pixels.
The values cannot be interpolated in this case and this will not be a proper testing for the centroid feature.

Reply:
inDataV1 is a fragment `input` variable. This is distinctly not a uniform, it is the interpolated output from the vertex-shader->fragment-shader

Interpolation functions require to be used with Fragment `input`'s

the value of the `input` variables rely on a vertex shader, they may be the same for everything, they may not.

Reply:
Oh. You are right. My bad. This is not a uniform.